Shopist/ShopistUITests/ShopistUITests.swift

```diff
@@ -44,6 +44,7 @@ class ShopistUITests: XCTestCase {
             app.terminate()
         }
         // Wait for uploading pending batches
+        app.launchToHomepage()
```

Comment:
Is it related to AF drop, or we fix something else here? 💭

Reply:
no it's unrelated actually

the idea behind that `Thread.sleep` was waiting for the app to upload pending batches, but i realized we sleep the thread while there is no app running.
at worst case, we might have been losing the latest batch to upload. otherwise it shouldn't be a major issue